lidarr/readme.md

```diff
@@ -1,5 +1,9 @@
 # README
 
+## Notice
+
+Please note that use of arr-scripts modified Lidarr's behavior and is not supported by the Lidarr community.
```

Comment:
Maybe change it to:

"Please note that use of arr-scripts is not supported by the Lidarr community.